benchmark/util/splice-one.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const common = require('../common');
+
+const bench = common.createBenchmark(main, {
+  n: [1e7],
+  size: [10, 100, 500],
+}, { flags: ['--expose-internals'] });
+
+function main({ n, size, type }) {
+  const { spliceOne } = require('internal/util');
```

Comment:
I'm curious, why put this here?

Reply:
The benchmark runs with `--expose-internals` so the main function has access to them but the outside doesn't.